index.js

```diff
@@ -31,7 +49,9 @@ function betweenExtremes() {}
  * Example: "A new month"
  * .- / -. . .-- / -- --- -. - ....
  */
-function morseCodeTranslator() {}
+function morseCodeTranslator(message, dictionary) {
+    return message.replace(' ','').split('').map(x => dictionary[x.toUpperCase()] || '').join(' ')
```

Comment:
in this case when we are chaining operations id actually recommend moving them to new lines.  This can help us break up the the logic a bit better.